frontend/src/components/RecommendedRecipes.js

```diff
@@ -21,16 +21,16 @@ import { Redirect } from "react-router-dom";
 import "./RecommendedRecipes.css";
 
 class RecommendedRecipes extends Component {
-  constructor(props) {
-    super(props);
+  constructor(properties) {
+    super(properties);
     this.state = {
       isLoading: true,
       isRedirect: false,
       recipes: [],
     };
   }
 
-  // retrieves recommended recipes from the back end 
+  // retrieves recommended recipes from the back end
   componentDidMount() {
```

Comment:
rename to make more clear

Reply:
*rename componentDidMount()

Reply:
It is a method of a Component class that gets called after render(). I cannot rename it https://reactjs.org/docs/react-component.html

Reply:
ohh my bad